railties/CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+*   Allow calling `bin/rails restart` outside of app directory.
+
+    The following would previously fail with a "No Rakefile found" error.
+
+    ```bash
+    BUNDLE_GEMFILE=blog/Gemfile blog/bin/rails restart
```

Comment:
Since [`bin/rails` requires `config/boot.rb`](https://github.com/rails/rails/blob/ef4841eac873ee20856944d129cdb14abf05bf5e/railties/lib/rails/generators/rails/app/templates/bin/rails.tt#L2) and [`config/boot.rb` sets `BUNDLE_GEMFILE`](https://github.com/rails/rails/blob/ef4841eac873ee20856944d129cdb14abf05bf5e/railties/lib/rails/generators/rails/app/templates/config/boot.rb.tt#L1), I think we can omit `BUNDLE_GEMFILE` here:

```suggestion
    blog/bin/rails restart
```

Reply:
Thanks. Yes this also working now without `BUNDLE_GEMFILE`.